scram-tools.file/tools/cepgen/cepgen.xml

```diff
@@ -0,0 +1,12 @@
+<tool name="cepgen" version="@TOOL_VERSION@">
+  <lib name="CepGen"/>
+  <lib name="CepGenHepMC2"/>
+  <lib name="CepGenLHAPDF"/>
+  <lib name="CepGenPythia6"/>
```

Comment:
@forthommel , if cepgen is building these AddOns then should `cepgen.spec` adds these as dependencies?

Reply:
Oh right! Sorry for the white noise I am introducing through this PR, as I am completely unfamiliar with this cmsdist integration.

Reply:
@smuzaffar would a scheme like this work?

Reply:
I have no idea how cepgen cmake works but if setting `TOOL_DIR` is enoght to tell cepgen cmake how to find an external package then this shold work

Reply:
by the way, do you want to link all of these libs together? e.g. when `cepgen` will be used in cmssw then do you want to have al these libs plus the libs from `lhapdf, hepmc, hepmc3, pythia6` link to cmssw lib? If not then I would suggest to create separate `cepgen-<tool>.xml` file e.g.

- cepgen-hepmc2.xml
```
<tool name="cepgen-hepmc2" version="@TOOL_VERSION@">
  <use name="cepgen"/>
  <use name="hepmc"/>
  <lib name="CepGenHepMC2"/>
</tool>
```

- cepgen-hepmc3.xml
```
<tool name="cepgen-hepmc3" version="@TOOL_VERSION@">
  <use name="cepgen"/>
  <use name="hepmc3"/>
  <lib name="CepGenHepMC3"/>
</tool>
```

and then one can use `cepgen-hepmc2`, `cepgen-hepmc3`, or `cepgen-lhapdf`

Reply:
Indeed, sorry for my lack of clarity!

The various CepGenHepMC2/CepGenPythia8/... libraries are indeed supposed to be used altogether with the main CepGen library in a EDProducer under development. The latter library is defining the main objects and their algorithmic parts, while the earlier are/can be loaded at runtime to extend the list of tools available (pretty much like any EDAnalyzer/ESSource/...).

And as a matter of fact the `cepgen` executable will not be used at all (and probably could be stripped off this PR? but it could possibly be a good unit test, if such things exist in cmsdist, that the main CepGen library+tool is well built).

Reply:
@forthommel , ok understood. Can you lease also add 
```
<use name="hepmc3"/>
<use name="hepmc"/>
<use name="lhapdf"/>
<use name="pythia6"/>
```